themes/sensei-course-theme/templates/lesson.html

```diff
@@ -50,7 +50,7 @@
         <!-- wp:post-title {"level": 1} /-->
         <!-- wp:sensei-lms/course-theme-notices /-->
         <!-- wp:post-content /-->
-        <!-- wp:sensei-lms/quiz-actions /-->
+        <!-- wp:sensei-lms/pagination /-->
```

Comment:
WDYT of `wp:sensei-lms/page-actions`, since it can have other actions to the page in the quiz?

Reply:
Sounds good, renamed it in [dde384f](https://github.com/Automattic/sensei/pull/4686/commits/dde384ff6a9cc22aa3995a34db2cc039727b1b7e).
Also added the block for this in the editor that I missed.